packages/pyright-internal/src/languageService/completionProvider.ts

```diff
@@ -1923,12 +1639,14 @@ export class CompletionProvider {
                     priorText,
                     priorWord,
                     postText,
-                    undefined /* TODO: base expression node */,
-                    parseNode,
                     completionList
                 );
                 return { completionList };
             }
+
+            if (this._tryAddTypedDictKeys(parseNode, priorText, postText, completionList)) {
```

Comment:
I don't know if this is the best place to add the completions but keeping the code in `_addLiteralValuesForTargetType` resulted in incorrect completions as the function is called from `_addLiteralValuesForArgument` which passes the signature type instead of the expected type at the current parse node